From d50fa39e332b08d08c4d312a44158bce192c62b7 Mon Sep 17 00:00:00 2001
From: Dan Gildea <dgildea>
Date: Wed, 28 Apr 2021 07:29:25 -0400
Subject: [PATCH] Fix #67 DESTDIR not fully implemented

install.pl was testing that it could create directory $TEXPATH
instead of $DESTDIR/$TEXPATH

remove test - directory creation will be tested by &install_file
---
 config/install.pl | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/config/install.pl b/config/install.pl
index b2f7c0e..d732176 100755
--- a/config/install.pl
+++ b/config/install.pl
@@ -373,13 +373,7 @@
 #-----------------------------------------------------------------------------
 
 if($cfg{TEXPATH}) {
-  print "\nNote: trying to install LaTeX2HTML style files in TeX directory tree\n     ($cfg{TEXPATH})\n";
-  unless(mkpath($cfg{TEXPATH})) {
-  #my $testpath = $cfg{TEXPATH}; # to strip (latex2)html
-  #$testpath =~ s/[$dd$dd][^$dd$dd]*$//;
-  #if((-d $cfg{TEXPATH} && !-w _) || (-d $testpath && !-w _)) {
-    print STDERR "\nError: Cannot install LaTeX2HTML style files in $cfg{TEXPATH}\n";
-  } else {
+    print "\nNote: trying to install LaTeX2HTML style files in TeX directory tree\n     ($cfg{TEXPATH})\n";
     my $dir = 'texinputs';
     my $dest = $cfg{TEXPATH};
     unless(opendir(DIR,$dir)) {
@@ -401,7 +395,6 @@
         system($cfg{MKTEXLSR});
       }
     }
-  }
 }
 print "Done. Have a lot of fun with LaTeX2HTML!\n";
 exit 0;
